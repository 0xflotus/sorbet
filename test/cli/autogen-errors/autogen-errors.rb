# typed: true
class Foo < Bar
end
class Bar < Foo
end
