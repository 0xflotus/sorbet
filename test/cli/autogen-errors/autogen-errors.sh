# shellcheck disable=SC2069
main/sorbet --silence-dev-message -p autogen-msgpack --stop-after=namer --skip-dsl-passes test/cli/autogen-errors/autogen-errors.rb 2>&1 1>/dev/null
