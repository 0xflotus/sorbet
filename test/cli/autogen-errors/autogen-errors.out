test/cli/autogen-errors/autogen-errors.rb:4: Circular dependency: `Bar` and `Foo` are declared as parents of each other http://go/e/5011
     4 |class Bar < Foo
                    ^^^
Errors: 1
